src/routes/TraderForm/Form/MarginAmountField/index.tsx

```diff
@@ -19,6 +20,9 @@ export const MarginAmountField: React.FunctionComponent<NotionalAmountProps> = (
   const infoPostSwap = useAppSelector(selectInfoPostSwap);
   const aMM = useAppSelector(selectSwapFormAMM);
 
+  const marginRequiredError = marginAmount.error !== null && marginAmount.error !== 'WLT';
+  const walletError = marginAmount.error === 'WLT';
```

Comment:
perhaps move this to selector as well.

Reply:
[753f5ee](https://github.com/Voltz-Protocol/voltz-ui/pull/576/commits/753f5ee4ee927fec2af52222bba233742d903d88)